.github/workflows/smoke.yml

```diff
@@ -0,0 +1,57 @@
+name: Smoke tests
+env:
+  IMAGE_REGISTRY: "localhost:5000"
+  TAG: "latest"
+  DOCKER_ORG: enmasse
+  KUBERNETES_NAMESPACE: "enmasse-infra"
+  SYSTEMTEST_PROFILE: smoke
+  SMOKE: true
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master      
+    
+    - name: setup-docker
+      uses: docker-practice/actions-setup-docker@0.0.1
+    
+    - name: Minikube setup with registry
+      uses: EnMasseProject/minikube-setup@V1.0.0
+      with:
+        k8s-version: 1.15.0
+    
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11.0.2
+
+    - name: Setup Ruby environment with ruby-build
+      uses: clupprich/ruby-build-action@v1.0.0
+  
+    - name: Install dependencies
+      run: ./.github/scripts/install_dep.sh
+    
+    - name: Setup OLM
+      run: ./.travis/setup-olm.sh
+
+    - name: Build
+      run: ./.github/scripts/build.sh
+
+    - name: Smoke tests
+      run: ./.travis/test.sh
```

Comment:
Perhaps copy this to .github/scripts (leave the original as is for now..) so that its all in the same place?